### PR TITLE
Remove executable as an argument to containers

### PR DESCRIPTION
**Pull request by zdicesare**
Jaeger containers were changed to no longer require the path
to the executable as an argument to docker run, so remove all
instances of executable path

Signed-off-by: Zachary DiCesare zachary.v.dicesare@gmail.com

### CODE REVIEW

**Comment by jpkrohling** [comment]
I added the WIP marker to the title for the same reason as https://github.com/jaegertracing/jaeger-kubernetes/pull/87#issuecomment-389445294

File: jenkins-ci/jaeger-agent-template.yml [hunk 1]
```diff
@@ -84,7 +84,6 @@ objects:
           image: jaegertracing/jaeger-agent:${IMAGE_VERSION}
```
Comment (jpkrohling) [inline jenkins-ci/jaeger-agent-template.yml]: You'll need to update the `IMAGE_VERSION` parameter as well

**Comment by zdicesare** [comment]
Build will fail until 1.6 images are out

**Comment by jkandasa** [comment]
@zdicesare @jpkrohling 

Collector service failed to start,
```
container_linux.go:247: starting container process caused "exec: \"--config-file=/conf/collector.yaml\": stat --config-file=/conf/collector.yaml: no such file or directory"
```
Query service failed to start,
```
container_linux.go:247: starting container process caused "exec: \"--config-file=/conf/query.yaml\": stat --config-file=/conf/query.yaml: no such file or directory"
```

**Comment by zdicesare** [comment]
Sorry about that @jkandasa, should be resolved now.

**Comment by jkandasa** [comment]
@zdicesare np. Thank you!
I see some test failures. I will check it and update here.

**Comment by jkandasa** [comment]
@jpkrohling @zdicesare Tested this on another OpenShift environment, all works well.
from test side LGTM!

**Pull request merged**